aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -235,10 +269,13 @@ module aptos_framework::aptos_account {
     /// Ensure that APT Primary FungibleStore exists (and create if it doesn't)
     inline fun ensure_primary_fungible_store_exists(owner: address): address {
         let store_addr = primary_fungible_store_address(owner);
-        if (fungible_asset::store_exists(store_addr)) {
-            store_addr
-        } else {
-            object::object_address(&primary_fungible_store::create_primary_store(owner, object::address_to_object<Metadata>(@aptos_fungible_asset)))
+        if (fungible_asset::store_exists(store_addr)) { store_addr }
```

Comment:
I don't think this is standard. The body should be on separate lines and else should come right after }

Reply:
yeah, we can add a configuration option[`prefer_one_line_in_short_branch_blk`] in the movefmt.toml configuration file. We track this issue on https://github.com/movebit/movefmt/issues/18